### DISCUSSION TITLE
Group Assignment #1 📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#3] [Assignments]
Due 4/13 at 2pm PST

Post the link to your group repository alongside the group members here to complete [group assignment #1](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week1/group_assignment/)

### Example submission

Albert, Yoh, Nina: https://github.com/albertkun/my-example-project/

**Comment by kmtsujimoto** [comment]
Falkenroth, James; Lau, Amanda; Tsujimoto, Kyle: https://github.com/jamesfalkenroth/healthcare

**Comment by ariscorpus** [comment]
Corpus, Aris, Lee Paige, Lee Emma https://github.com/ariscorpus/stopasianhate

**Comment by SageLuong** [comment]
Cruz, Daniel, Luong, Sage, Wan, Dylan: https://github.com/DylanJWan/AA191ATransportation.git

**Comment by czhangy** [comment]
Delk, Victoria; Pakin, Zachary; Zhang, Charles: https://github.com/czhangy/aa191a-inequality

**Comment by danigold1020** [comment]
Alcantar, Angelica; Goldwirth, Danielle: [https://github.com/danigold1020/asianam-191a-oceanpollution](https://github.com/danigold1020/asianam-191a-oceanpollution)

**Comment by rebeccaksuen** [comment]
Elsa Mai, Rebecca Suen: (https://github.com/rebeccaksuen/Educational-Attainment)

**Comment by qatle** [comment]
Stella Kang, Trinh Le, Cassidy Tu: https://github.com/trinhqale/AM191A-group-project

**Comment by cassiuscarandang** [comment]
Cassius Carandang, Mohammed Azed, Josy Law: https://github.com/cassiuscarandang/aashousingandhomelessness

**Comment by catherinehhu** [comment]
Catherine Hu, Maizah Ali, Jonathan Lopez: https://github.com/catherinehhu/asian-am-191a-project

**Comment by mandymwlai** [comment]
Mandy Lai, Joanne Luong, Alana Malika https://github.com/joanneluong/AAS191A-SEA/blob/main/README.md

**Reply by ariscorpus** [reply]
Aris Corpus, Paige Lee, Emma Lee - https://github.com/ariscorpus/foodinsecurity
